docs/advanced_usage/profiling.rst

```diff
@@ -0,0 +1,99 @@
+Profiling
+*********
+
+Overview
+========
+
+Profiling your test methods helps you analyze performance and optimize efficiency.
+The `@profile` decorator makes it easy to profile your test methods and automatically
+generates a result file based on the test module name and test name if no file path is provided.
+
+Using the `@profile` Decorator
+==============================
+
+The `@profile` decorator can be used in two ways:
+- **Default usage**: Generates an uncompressed result file named after the test module and test name in your current working directory.
+- **Specifying** a file path**: Generates an uncompressed result file with the specified file path.
+- **Compressed output**: When `compress=True` is set, the output file is compressed into a `.cvf` format.
+
+Example Usage
+-------------
+
+```python
+from pykiso.profiling import profile
+
+@profile
+def test_method():
+    pass
+
+@profile
+def test_method("result.json"):
```

Comment:
"result.json" should be in the decorator argument.

Reply:
fixed